h2o-py/h2o/h2o.py

```diff
@@ -294,6 +294,9 @@ def get_mem_size(mmint, mmgb):
         if not start_h2o: raise
         if ip and not (ip == "localhost" or ip == "127.0.0.1"):
             raise H2OConnectionError('Can only start H2O launcher if IP address is localhost.')
+        if https:
+            raise H2OConnectionError('Unable to start local server with https enabled. Consider disabling https or'
```

Comment:
I don't line the wording of the error message. To me it suggest that we tried to start but were unable because of some internal issue/bug not user/config error. But I am having hard time coming up with better wording. Please help @angela0xdata

Reply:
How about "Starting local server is not available with https enabled"?

Reply:
@neemah2o love it!

Reply:
Great, I'll make it so.

Reply:
Done. Thanks for the suggestions.